### PR TITLE
Fix styling to conform to style guide

### PR DESCRIPTION
**Pull request by deejes**
No changes to functionality, only style changes.

### CODE REVIEW

**Review by shreymahendru — COMMENTED** [review]
Remove package-lock.json

File: src/foam/nanos/auth/Address.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ foam.CLASS({
     {
       class: 'String',
       name: 'address1',
-      //required: true
+      // required: true
```
Comment (shreymahendru) [inline src/foam/nanos/auth/Address.js:48]: remove the commented out code if it's no  `required`

File: src/foam/nanos/auth/Country.js [hunk 2]
```diff
@@ -28,7 +28,9 @@ foam.CLASS({
     name: 'toString',
     returns: 'String',
     javaReturns: 'String',
-    code: function() { return "Country: " + this.code + ", " + this.name; },
+    code: function() {
+      return 'Country: ' + this.code + ', ' + this.name;
```
Comment (shreymahendru) [inline src/foam/nanos/auth/Country.js:32]: User string interpolation here.

Comment (kgrgreer) [reply src/foam/nanos/auth/Country.js:32]: Interpolation used to be about 100 times slower, but I just ran a benchmark and it seems that interpolation is as fast or faster now.

Comment (kgrgreer) [reply src/foam/nanos/auth/Country.js:32]: But I wouldn't bother changing existing code to use interpolation.

**Pull request merged**